### PR TITLE
Fix test_installation to skip using parallel_gpu_execute when not using gpu.

### PR DESCRIPTION
**Pull request by gunan**

### CODE REVIEW

**Comment by gunan** [comment]
@caisq  How do we run pip tests for this one?

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
debugging, please do not merge.

**Comment by gunan** [comment]
It is finally working for CPU tests, but please wait until I complete testing on all platforms.

File: tensorflow/tools/ci_build/builds/test_installation.sh [hunk 1]
```diff
@@ -411,21 +410,27 @@ SKIP_COUNTER=0
...
-  N_JOBS=8
-  echo "Cannot determine the number of processors"
-  echo "Using default concurrent job counter ${N_JOBS}"
-fi
-
-if [[ ! -z "${TF_BUILD_SERIAL_INSTALL_TESTS}" ]] &&
-   [[ "${TF_BUILD_SERIAL_INSTALL_TESTS}" != "0" ]]; then
+if [[ "${TF_BUILD_USE_GPU}" == "1" ]]; then
   N_JOBS=$TF_GPU_COUNT
+  TEST_RUN_COMMAND_PREFIX="${SCRIPT_DIR}/../gpu_build/parallel_gpu_execute.sh" \
```
Comment (yifeif) [inline tensorflow/tools/ci_build/builds/test_installation.sh]: Do we still need these?

**Review by yifeif — APPROVED** [review]

**Pull request merged**